src/apiCalls.js

```diff
@@ -0,0 +1,49 @@
+const apiEndpoint = 'https://rancid-tomatillos.herokuapp.com/api/v2'
+
+function loadMovies() {
+  return fetch(`${apiEndpoint}/movies`)
+    .then(response => response.json())
+}
+
+function loadSingleMovie(id) {
+  return fetch(`${apiEndpoint}/movies/${id}`)
+    .then(response => response.json())
+}
+
+function loadSingleMovieVideo(props) {
+  return fetch(`${apiEndpoint}/movies/${props.id}/videos`)
+    .then(response => response.json())
+}
+
+function loginUser(props) {
```

Comment:
You are having multiple users? Is this your additional feature for iteration 5? If so, you need to DM me and Robbie to get it approved first.